### PR TITLE
Issue 583

### PR DESCRIPTION
**Pull request by brucehoff**

### CODE REVIEW

**Comment by brucehoff** [comment]
https://github.com/Sage-Bionetworks/synapseDocs/issues/583

File: articles/challenge_administration.md [hunk 1]
```diff
@@ -65,6 +65,18 @@ Challenge participants can submit Synapse Entities (e.g. `File`, `Folder`, `Proj
 
 Please visit the [Evaluation Queue article](evaluation_queues.md) to learn more about queue configuration.
 
+One of the features of Synapse for DREAM Challenges is the live compilation of submission statistics for all evaluation queues.  You can see the stat's for various challenges [here](https://www.synapse.org/#!Synapse:syn2504723/wiki/65150).  In order to activate statistics for your evaluation queues:
```
Comment (kdaily) [inline articles/challenge_administration.md]:
```suggestion
One of the features of Synapse for DREAM Challenges is the live compilation of submission statistics for all evaluation queues.  You can see the statistics for various challenges [here](https://www.synapse.org/#!Synapse:syn2504723/wiki/65150).  In order to activate statistics for your evaluation queues:
```

File: articles/challenge_administration.md [hunk 2]
```diff
@@ -65,6 +65,19 @@ Challenge participants can submit Synapse Entities (e.g. `File`, `Folder`, `Proj
...
+1. Navigate to your Challenge project in Synapse.
+1. Click on the Challenge tab.
+1. Next to each Evaluation submission queue, click Share.
+1. In the name field type `evaluationstatistics`. The new entry will appear in the list of accessors above.
+1. Next to the new entry use the pull down menu to change Can View to Can Score.
+1. Click Save.
+
+Statistics are updated weekly. They are also retroactive - you do not have to enable statistics at the beginning of your challenge to have the entire history reported.
+
+<br/>
```
Comment (kdaily) [inline articles/challenge_administration.md]:
```suggestion
```

File: articles/challenge_administration.md [hunk 3]
```diff
@@ -65,6 +65,19 @@ Challenge participants can submit Synapse Entities (e.g. `File`, `Folder`, `Proj
...
+
+1. Navigate to your Challenge project in Synapse.
+1. Click on the Challenge tab.
+1. Next to each Evaluation submission queue, click Share.
+1. In the name field type `evaluationstatistics`. The new entry will appear in the list of accessors above.
+1. Next to the new entry use the pull down menu to change Can View to Can Score.
+1. Click Save.
+
+Statistics are updated weekly. They are also retroactive - you do not have to enable statistics at the beginning of your challenge to have the entire history reported.
+
```
Comment (kdaily) [inline articles/challenge_administration.md]:
```suggestion
```

**Review by kdaily — CHANGES_REQUESTED** [review]
Minor changes, but looks good overall!

Please consider adding in a short section that says what statistics they will get by doing this, instead of relying on the user going to the page to see an example.

**Comment by brucehoff** [comment]
I made the requested changes.  Kindly review and merge.

**Review by thomasyu888 — COMMENTED** [review]
I really like this, but the challenge statistics page is not a public page.  I wonder if we need to also add a short description of how someone would gain access to this page.

File: articles/challenge_administration.md [hunk 4]
```diff
@@ -65,6 +65,17 @@ Challenge participants can submit Synapse Entities (e.g. `File`, `Folder`, `Proj
 
 Please visit the [Evaluation Queue article](evaluation_queues.md) to learn more about queue configuration.
 
+One of the features of Synapse for DREAM Challenges is the live compilation of submission statistics for all evaluation queues, including total submission count, count per individual/team, count per submission state (scored, invalid) and count per week.  You can see the statistics for various challenges [here](https://www.synapse.org/#!Synapse:syn2504723/wiki/65150).  In order to activate statistics for your evaluation queues:
```
Comment (kdaily) [inline articles/challenge_administration.md]: Per @thomasyu888:
```suggestion
One of the features of Synapse for DREAM Challenges is the live compilation of submission statistics for all evaluation queues, including total submission count, count per individual/team, count per submission state (scored, invalid) and count per week.  You can see the statistics for various challenges [here](https://www.synapse.org/#!Synapse:syn2504723/wiki/65150).  In order to activate statistics for your evaluation queues, you must be an administrator of the challenge Project. Each queue needs to be configured to generate the statistics. To do this:
```

**Comment by brucehoff** [comment]
@thomasyu888 I made the page visible to all logged-in users.

File: articles/challenge_administration.md (hunk 4 above)
Comment (kdaily) [reply articles/challenge_administration.md]: @brucehoff can you review and merge this suggested change? I cannot do so. Thank you!

**Review by kdaily — APPROVED** [review]
LG!

**Pull request merged**